### ISSUE TITLE
Add a "welcome page" after you installed the addon

### ISSUE THREAD

**Issue by Sav22999** [Mte90/Share-Backported#160]
In which you could insert some information (link to GitHub, and FAQ)

**Comment by Ryuno-Ki** [comment]
What would be needed in the [manifest.json](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json)?
(Perhaps link to an extension where you observed the behaviour, so I could take a look).

**Comment by Mte90** [comment]
I don't think that there is a permission for that but we should implement something.

**Comment by Ryuno-Ki** [comment]
I wouldn't know how to know when to trigger the opening of a new page as welcome screen.
I think, I've saw it somewhere, but can't recall the add-on off-hand.

**Comment by Mte90** [comment]
The way to do it is to check if there are any settings, if it is missing means that is a new installation.

**Comment by Mte90** [comment]
There is an API now https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/onInstalled

**Comment by Mte90** [comment]
I was thinking to add also a text about some tools like telegram or whatsapp that open the client installed in the machine as alert in that page.

**Comment by SameerSahu007** [comment]
we can redirect user to a welcome webpage when someone installs our add on?

**Comment by Mte90** [comment]
Yes this is the purpose of this ticket, we just need to do it

**Comment by SameerSahu007** [comment]
great I can work on it
